### PR TITLE
[10.0]Sale Migration missing tuples

### PR DESCRIPTION
**Pull request by flotho**
Implements tuple syntax

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Next time, you can make directly the PR and put the issue here.

File: addons/sale/migrations/10.0.1.0/pre-migration.py [hunk 1]
```diff
@@ -92,20 +92,20 @@ def warning_update_module_names_partial(cr):
         SELECT id
         FROM ir_model_fields
         WHERE model IN ('res.partner', 'product.template') AND name in %s
-    """, (moved_fields,))
+    """, (tuple(moved_fields),))
     field_ids = [r[0] for r in cr.fetchall()]
```
Comment (pedrobaeza) [inline addons/sale/migrations/10.0.1.0/pre-migration.py]: Better to make here `fields_ids = tuple([r[0]...` and you avoid to put it in all the places where it's used.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please check if there's the same case on the rest of the script and the post-migration script

It was not detected by Travis as the module is not installed being removed. @hbrunn can we make something with this to recognize also |del| modules to be installed on version n-1?

File: addons/sale/migrations/10.0.1.0/pre-migration.py (hunk 1 above)
Comment (hbrunn) [reply addons/sale/migrations/10.0.1.0/pre-migration.py]: this check happpens in https://github.com/OCA/OpenUpgrade/blob/10.0/.travis.yml#L64 - so either use one of the keywords this greps for deleted addons too or change the regex to output also deleted addons

**Comment by flotho** [comment]
Hi @pedrobaeza, sorry to ask such an obvious thing but I'm not able to identify what part of code make this job failed :  https://travis-ci.org/OCA/OpenUpgrade/jobs/250747464#L387

BTW, I'm not sure to understand the #958, is it better to use tuple or change to ANY ?

Regards

**Comment by pedrobaeza** [comment]
It's because:
```
./addons/sale/migrations/10.0.1.0/pre-migration.py:89:66: W291 trailing whitespace
```
You have an space at the end of the line. Configure your IDE to remove them automatically.

About the best query, `IN` or `= ANY`, the final conclusion is that it performs the same (IN is transformed to = ANY), so the only advantage of ANY is to not convert everything to tuples.

**Comment by flotho** [comment]
@pedrobaeza , thanks I was expecting to have the error below the red line.

**Comment by flotho** [comment]
Hi @pedrobaeza , @hbrunn , 

Is it possible to have either a merge on my PR or a positive review and merge on the #958 ?

regards

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**